packages/api/src/services/save_page.ts

```diff
@@ -139,14 +143,37 @@ export const savePage = async (
       }
     }
   } else {
-    if (!(await createPage(articleToSave, ctx))) {
+    pageId = await createPage(articleToSave, ctx)
+    if (!pageId) {
       return {
         errorCodes: [SaveErrorCode.Unknown],
         message: 'Failed to create new page',
       }
     }
   }
 
+  if (pageId && parseResult.highlightData) {
+    const highlight = {
+      updatedAt: new Date(),
+      createdAt: new Date(),
+      userId: ctx.uid,
+      elasticPageId: pageId,
```

Comment:
We used to have this in the database and we might not need it anymore

Reply:
I mean the property `elasticPageId`

Reply:
Oh, so its like a back link from the highlight to the page? It does look like its being set now for other highlights

Reply:
Oh then maybe we better keep it